dashboard/tests/test_helpers.py

```diff
@@ -20,3 +20,27 @@ def test_shorten_node_key_none(self):
             None,
             shorten_node_key(None)
         )
+
+    def test_get_natural_size_1byte(self):
+        self.assertEqual(
+            '1 Byte',
+            get_natural_size(1)
+        )
+
+    def test_get_natural_size_1kb(self):
+        self.assertEqual(
+            '1.00 KB',
+            get_natural_size(1024)
+        )
+
+    def test_get_natural_size_1mb(self):
+        self.assertEqual(
+            '1.00 MB',
+            get_natural_size(1024 * 1024)
+        )
+
+    def test_get_natural_size_1gb(self):
+        self.assertEqual(
+            '1.00 GB',
+            get_natural_size(1024 * 1024 * 1024)
+        )
```

Comment:
I would merge all these test method into a single `test_get_natural_size` - splitting them doesn't seem to give much.